src/components/InputItem.tsx

```diff
@@ -93,7 +93,8 @@ export class InputItem extends PureComponent<Props, State> {
 
   render() {
     const { isAnimatedFocused, isActive } = this.state;
-    const { label, suffix, error } = this.props;
+    const { label, suffix, error, secureTextEntry } = this.props;
```

Comment:
add secureTextEntry to props, create a type for it something like `type secureTextEntryType = 'default' | 'visible-password'`